packages/main/src/TabContainer.js

```diff
@@ -131,6 +146,7 @@ const metadata = {
 		 * @type {boolean}
 		 * @defaultvalue false
 		 * @public
+		 * @deprecated
```

Comment:
This is a breaking change. It won't break existing code, but it will render a property useless. Not sure if this qualifies for `1.x.x`, please consult with POs. At least, there should be a deprecation explanation, telling users which other APIs to use to achieve the same behavior as before, if at all possible.